src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -390,7 +608,9 @@ export const getInitialInputValues = (offerQuote: OfferQuote) =>
       () =>
         getInitialNorwegianHomeContentValues(
           offerQuote.id,
-          offerQuote.quoteDetails as NorwegianHomeContentsDetails,
+          (offerQuote.quoteDetails as QueryQuoteDetails) as QueryQuoteDetails & {
+            __typename: 'NorwegianHomeContentsDetails'
```

Comment:
This double type cast looks weird to me but maybe it's required? Idk

Reply:
I have to cast it, but you can cast it to unknown/any but I thought it might be better to be more exact.